### PR TITLE
Add manhattan.rst and manhattan.sg

### PR DESCRIPTION
**Pull request by IOcodegeass**

### CODE REVIEW

**Comment by karlnapf** [comment]
If you update the data version that I just merged then the integration test will be executed

File: doc/cookbook/source/examples/distance/manhattan.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+==================
+Manhattan Distance
+==================
+
+The Manhattan distance for real valued features is the absolute differences between the feature dimensions of two data points.
```
Comment (karlnapf) [inline doc/cookbook/source/examples/distance/manhattan.rst]: is the absolute difference (singular)

Comment (karlnapf) [inline doc/cookbook/source/examples/distance/manhattan.rst]: between components of two data points

Comment (karlnapf) [inline doc/cookbook/source/examples/distance/manhattan.rst]: Please also say that is is L1 distance, and add a wiki reference below

**Comment by karlnapf** [comment]
Commit message: "Add Manhattan distance example and cookbook"

**Comment by IOcodegeass** [comment]
@karlnapf changes have been made. Kindly review them :)

**Comment by karlnapf** [comment]
Great! If you could now please squash the commits into a single one, we are ready to merge it

**Comment by IOcodegeass** [comment]
@karlnapf i have squashed previous commits into single one.

**Pull request merged**